packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/UpdateSliceZoneModalList.tsx

```diff
@@ -18,79 +16,27 @@ const UpdateSliceZoneModalList: React.FC<{
       <Grid
         gridTemplateMinPx="200px"
         elems={availableSlices}
-        defineElementKey={(slice: ComponentUI) => slice.model.name}
-        renderElem={(slice: ComponentUI) => {
-          const isPlaceholderSlice = placeholderSlices?.find(
-            (s) => s.model.id === slice.model.id
+        defineElementKey={(slice) => `${slice.from}-${slice.model.name}`}
+        renderElem={(slice) => {
+          const isPlaceholderSlice =
+            placeholderSlices?.some((s) => s.model.id === slice.model.id) ??
+            false;
+          const isInSliceZone = values.sliceKeys.includes(slice.model.id);
+          return (
+            <SharedSliceSelectionCard
+              isComingSoon={isPlaceholderSlice}
```

Comment:
❓ Hum for these kinds of props (isComingSoon, isSliceTemplate). I tough you prefer to not have it by props but by exporting component that can be reuse or not?

Here for example. Coming soon was something made to be temporary, and it's design like something built in not generic.

Reply:
What I'm not sure is you're bringing "down" back something that is specific for this component. But the "shared" component is then shared super far away from here and we could mix match the usage no?